modVM/src/org/aion/vm/BulkExecutor.java

```diff
@@ -124,7 +123,7 @@ public BulkExecutor(
      * @param logger The logger.
      * @param work The post-execution work to apply after each transaction is run.
      */
-    public BulkExecutor (
+    public BulkExecutor(
```

Comment:
formatting

Reply:
I don't think that's a formatting mistake. That's the result of applying the formatter, lol.

Reply:
ups, sorry about that!